cpu/esp_common/Makefile.include

```diff
@@ -104,14 +51,10 @@ ifneq (,$(filter esp_qemu,$(USEMODULE)))
   CFLAGS += -DQEMU
 endif
 
+# NOTE: These checks can be turned into normal conditionals when #9913 is fixed
 # use 32 priority levels if any WiFi interface or the ETH interface is used
-ifneq (,$(filter esp_wifi esp_now esp_eth,$(USEMODULE)))
-  CFLAGS += -DSCHED_PRIO_LEVELS=32
-endif
-
-ifneq (,$(filter esp_wifi esp_now esp_eth,$(USEMODULE)))
-  INCLUDES += -I$(RIOTCPU)/esp_common/esp-wifi
-endif
+CFLAGS += $(if $(filter esp_wifi_any esp_eth,$(USEMODULE)),-DSCHED_PRIO_LEVELS=32)
+INCLUDES += $(if $(filter esp_wifi_any esp_eth,$(USEMODULE)),-I$(RIOTCPU)/esp_common/esp-wifi)
```

Comment:
It enough to add this include for `esp_wifi`.
```suggestion
INCLUDES += $(if $(filter esp_wifi,$(USEMODULE)),-I$(RIOTCPU)/esp_common/esp-wifi)
```

Reply:
yes? previously it was also checking for `esp_eth` and `esp_now`

Reply:
I know, but once we are cleaning up ... Even more, it seems that it can be completely removed.
```suggestion
```

Reply:
> Even more, it seems that it can be completely removed.

Why is this? Who is adding the path?

Reply:
No one. The header files in the directory are only used by files in same directory.

Reply:
> No one. The header files in the directory are only used by files in same directory.

Probably, it was just a leftover.

Reply:
Removed